packages/fakta-fordel-beregningsgrunnlag/src/components/refusjon/VurderEndringRefusjonRad.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  dateFormat, hasValidDate, required, dateAfterOrEqual,
+} from '@fpsak-frontend/utils';
+import { Column, Row } from 'nav-frontend-grid';
+import { DatepickerField } from '@fpsak-frontend/form';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Normaltekst } from 'nav-frontend-typografi';
+import { refusjonAndelTilVurderingPropType } from '../../propTypes/beregningsgrunnlagPropType';
+import styles from './vurderEndringRefusjonRad.less';
+
+const visningsnavn = (andel) => {
+  if (andel.arbeidsgiverNavn) {
+    return andel.arbeidsgiverNavn;
+  } if (andel.arbeidsgiverId && andel.arbeidsgiverId.arbeidsgiverOrgnr) {
+    return andel.arbeidsgiverId.arbeidsgiverOrgnr;
+  } if (andel.arbeidsgiverId && andel.arbeidsgiverId.arbeidsgiverAktørId) {
+    return andel.arbeidsgiverId.arbeidsgiverAktørId;
+  }
+  return undefined;
+};
+
+const FIELD_KEY = 'REFUSJON_ENDRING_DATO';
+
+export const lagNøkkel = (andel) => `${FIELD_KEY}${andel.arbeidsgiverNavn}${andel.aktivitetStatus.kode}${andel.internArbeidsforholdRef}
+${andel.arbeidsgiverId.arbeidsgiverOrgnr}${andel.arbeidsgiverId.arbeidsgiverAktørId}`;
+
+export const VurderEndringRefusjonRad = ({
+  refusjonAndel,
+  readOnly,
+  skjæringstidspunkt,
+}) => {
+  if (!refusjonAndel) {
+    return null;
+  }
+  const navn = visningsnavn(refusjonAndel);
+  return (
+    <>
+      <Row>
+        <Column xs="8">
+          <FormattedMessage
+            id="BeregningInfoPanel.RefusjonBG.Arbeidsgiver"
+            values={{
+              ag: navn,
+              dato: dateFormat(refusjonAndel.nyttRefusjonskravFom),
+              b: (...chunks) => <b>{chunks}</b>,
```

Comment:
Kva er dette?

Reply:
Prøvde i lang tid å få til FormattedMessage med fet tekst rundt en variabel men fikk ikke til. Dette er slik det er gjort i OmsorgFaktaForm.jsx. Tror problemet er at vi har gått bort fra FormattedHTMLMessage og for å få til dette med vanlig FormattedMessage til å støtte er dette syntaksen

Reply:
Ok, skjønte ikkje heilt syntaksen, men visst det fungerer er jo det bra.